gemini/sample-apps/fashion-trends-prediction/app/pages/0_ Trend Prediction.py

```diff
@@ -0,0 +1,405 @@
+                                        padding: 120px;
+                                        border-radius: 5px;
+                                        background-color: #f5f5f5;
+                                    }
+                                    </style>
+                                    <div class="box-container">
+                                        Image generation failed due to responsible AI restrictions.
+                                    </div>
+                                    """,
+                            unsafe_allow_html=True,
+                        )
+                        print("error in generating imgs: ", error)
+
+                st.write("")
+                if len(state["additional_outfits"][selected_button]) > 0:
+                    st.write("##### Similar: ")
+
+                for item in state["additional_outfits"][selected_button]:
+                    st.markdown(f" - {item}")
+
+                if country == "India":
+                    st.write("#### Relevant Articles")
+                    none_found = True
+
+                    if "relevant_articles" not in state[outfit]:
+                        with st.spinner("Fetching relevant articles..."):
+                            print(state["items"][selected_button])
+                            state[outfit]["relevant_articles"] = articles.getArticles(
+                                state["items"][selected_button][0].lower()
+                            )
+
+                    for article in state[outfit]["relevant_articles"]:
+                        st.write(article[0])
+                        st.link_button("Vogue Link", article[1])
+                        none_found = False
+
+                    if none_found:
+                        st.write("No relevant articles found")
+
+            else:
+                st.markdown(
+                    """
+                            <style>
+                            div.stMarkdown div.css-17eq0hr {
+                                text-align: center;
+                            }
+                            </style>
+                            """,
+                    unsafe_allow_html=True,
+                )
+                st.write("###### Select an item to view details")
+
+    with outer_tabs[1]:
+        response = state["summary"]
+        details_content = ""
+        for index, (key, values) in enumerate(response.items()):
+            values = [value for idx, value in enumerate(values, start=1)]
+            print("values: ", values)
+            values = [v.capitalize() for v in values]
+            print("values: ", values)
+            values_str = ",&nbsp;&nbsp;&nbsp;".join(
+                [f"{idx}. {value}" for idx, value in enumerate(values, start=1)]
+            )
+            print("values_str: ", values_str)
+            if index != len(response) - 1:
+                details_content += f"""
```

Comment:
Could you load these html snippets into an html file and import them instead of hardcoding them into the python? You could use jinja templates. https://jinja.palletsprojects.com/en/3.1.x/